packages/shared/src/server/queries/clickhouse-filter/clickhouse-filter.ts

```diff
@@ -84,9 +96,10 @@ export class NumberFilter implements Filter {
   }
 
   apply(): ClickhouseFilter {
-    const varName = `stringFilter${this.field}`;
+    const uid = randomCharacters();
```

Comment:
Would `crypto.randomBytes(4).toString('hex')` work as well? You don't need the full set of chars, right?
Here, you'd get a string like `c9a6fff4`.

Reply:
I would want to have characters only without numbers.

Reply:
@maxdeichmann Ehm, but you have numbers in your code snippet above? 😄

Reply:
removed. thx.